### PR TITLE
 RavenDB-8678 PR review changes

### PR DESCRIPTION
**Pull request by fitzchak**

### CODE REVIEW

File: src/Raven.Server/Documents/Expiration/ExpirationStorage.cs [hunk 1]
```diff
@@ -32,12 +32,22 @@ public ExpirationStorage(DocumentDatabase database, Transaction tx)
             tx.CreateTree(DocumentsByExpiration);
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```
Comment (ayende) [inline src/Raven.Server/Documents/Expiration/ExpirationStorage.cs]: Drop this, no need to do this.

File: src/Raven.Server/Documents/Expiration/ExpirationStorage.cs [hunk 2]
```diff
@@ -32,12 +32,22 @@ public ExpirationStorage(DocumentDatabase database, Transaction tx)
...
             if (document.TryGet(Constants.Documents.Metadata.Key, out BlittableJsonReaderObject metadata) == false ||
                 metadata.TryGet(Constants.Documents.Metadata.Expires, out string expirationDate) == false)
                 return;
 
+            PutInternal(context, lowerId, document);
+        }
+
+        private void PutInternal(DocumentsOperationContext context, Slice lowerId, BlittableJsonReaderObject document)
+        {
+            if (document.TryGet(Constants.Documents.Metadata.Key, out BlittableJsonReaderObject metadata) == false ||
```
Comment (ayende) [inline src/Raven.Server/Documents/Expiration/ExpirationStorage.cs]: Why do double the work? Make this method accept the expiration date as parameter.

**Review by ayende — CHANGES_REQUESTED** [review]

File: src/Raven.Server/Documents/Expiration/ExpirationStorage.cs (hunk 2 above)
Comment (fitzchak) [reply src/Raven.Server/Documents/Expiration/ExpirationStorage.cs]: Fixed

File: src/Raven.Server/Documents/Expiration/ExpirationStorage.cs (hunk 1 above)
Comment (fitzchak) [reply src/Raven.Server/Documents/Expiration/ExpirationStorage.cs]: OK

**Pull request merged**